### PR TITLE
Feature/vmo 6508/localize backend validations

### PR DESCRIPTION
**Pull request by safydy-r**
Changes:
- disable storyshots for now (added a TODO to put it back)
- do not dispaly duplicated errors from backend

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -50,6 +50,7 @@
     "bootstrap": "^4.5.3",
     "bootstrap-datetimepicker": "^0.0.7",
     "bootstrap-vue": "^2.21.1",
+    "caniuse-lite": "^1.0.30001377",
```
Comment (safydy-r) [inline package.json:53]: I tried to make the test working

File: src/store/validation/index.ts [hunk 2]
```diff
@@ -151,10 +151,11 @@ export const actions: ActionTree<IValidationState, IRootState> = {
     Object.keys(backendErrorsList).forEach((currentUuid) => {
       const key = `backend/${type}/${currentUuid}`
       const currentErrors = backendErrorsList[currentUuid]
+      const uniqueErrors = getUniqueObjectValuesFromArray(currentErrors, 'message') as { message: string }[]
```
Comment (object-required) [inline src/store/validation/index.ts:154]: Would it make sense to use `_.uniqBy` which seems to do the same thing?

**Review by object-required — APPROVED** [review]

File: tests/unit/utils/validationHelpers.spec.ts [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+describe('getUniqueObjectValuesFromArray', () => {
+  it('should return unique values', () => {
+    const listWithDuplicates = [{message: 'error 1'}, {message: 'error 1'}, {message: 'error 2'}]
+    const withoutDuplicates = getUniqueObjectValuesFromArray(listWithDuplicates, 'message')
+    expect(withoutDuplicates.sort())
+      .toEqual([{message: 'error 1'}, {message: 'error 2'}].sort())
+  })
+
+  it('should return empty array if undefined', () => {
```
Comment (bulatgab) [inline tests/unit/utils/validationHelpers.spec.ts:11]: if empty

File: tests/unit/utils/validationHelpers.spec.ts [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+      .toEqual([])
+  })
+
+  it('should return same array if no key provided', () => {
+    const listWithDuplicates = [{message: 'error 1'}, {message: 'error 1'}, {message: 'error 2'}]
+    const withoutDuplicates = getUniqueObjectValuesFromArray(listWithDuplicates, '')
+    expect(withoutDuplicates)
+      .toEqual(listWithDuplicates)
+  })
+})
```
Comment (bulatgab) [inline tests/unit/utils/validationHelpers.spec.ts:24]: unit tests ❤️

**Comment by bulatgab** [comment]
Just to clarify, does this resolve the ticket (VMO-6508), i.e. the localization issue? In other words, was it caused by the non-uniqueness of validation messages? Or is it fixed somewhere else?

**Review by bulatgab — APPROVED** [review]

File: src/store/validation/index.ts (hunk 2 above)
Comment (safydy-r) [reply src/store/validation/index.ts:154]: Exactly ! I've been looking for this yesterday, hahaha. 🤪 😝 😹

**Pull request merged**